apps/generator/test/integration.test.js

```diff
@@ -62,8 +62,11 @@ describe('Integration testing generateFromFile() to make sure the result of the
       templateParams: { version: 'v1', mode: 'production' }
     });
     await generator.generateFromFile(dummySpecPath);
-    const file = await readFile(path.join(outputDir, testOutputFile), 'utf8');
-    expect(file).toMatchSnapshot();
+    const mdFile = await readFile(path.join(outputDir, testOutputFile), 'utf8');
+    //react template has hooks lib enabled and generation of asyncapi document that was passed as input should work out of the box without adding @asyncapi/generator-hooks to dependencies
+    const asyncAPIFile = await readFile(path.join(outputDir, 'asyncapi.yaml'), 'utf8');
```

Comment:
basically test is extended and making sure asyncapi document also gets into output because the proper hook (without adding it to dependency) is configured

not sure it makes sense to validate in test, if it really is not added to dependencies 🤔

Reply:
> not sure it makes sense to validate in test, if it really is not added to dependencies 🤔
not sure why we need to check for asyncapi document is in output ?

Reply:
already answered in a meeting

for others: it is needed as before this PR, and in general, by default, generator do not output asyncapi document that it received in input. There is this generic hook for it, and it is added to our test template with this PR - so we need to make sure in test we see it in output - to confirm hooks integation works